meta-psa/recipes-security/mbed-crypto/mbed-crypto_1.0.0.bb

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+HOMEPAGE = "https://github.com/ARMmbed/mbed-crypto"
+DESCRIPTION = "Mbed Crypto is a reference implementation of the Arm PSA Crypto API "
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = " \
+    file://LICENSE;md5=302d50a6369f5f22efdb674db908167a \
+    file://apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+"
+
+SECTION = "libs"
+
+SRC_URI = "https://github.com/ARMmbed/mbed-crypto/archive/mbedcrypto-${PV}.tar.gz"
+SRC_URI[md5sum] = "06dd48905c236f7939d03b09bcf7f1a2"
+SRC_URI[sha256sum] = "70b3bde55c943b931fd296bba3936fa677416b3997aac10ed3ba193ba16e8037"
+
+S = "${WORKDIR}/${BPN}-mbedcrypto-${PV}"
+
+# We add a '-test' package here, but ideally we'd be using the ptest framework.
+# MBL has an issue where enabling the ptest DISTRO-FEATURE results in
+# dependency cycles (IOTMBL-2167), so leave ptest for later.
```

Comment:
I don't like the idea of adding jira tickets ids in the source code.

Reply:
Why not?

Reply:
Discussed IRL with @diego-sueiro and @jjohnson-arm. I'll create a GitHub issue for our ptest problem and reference that in the code instead.

Reply:
https://github.com/ARMmbed/meta-mbl/pull/576/commits/5573ff0733f690ee80e5b1a3871af0a84260c353

Reply:
Created https://github.com/ARMmbed/meta-mbl/issues/577.